src/main/java/com/sugarcrm/candybean/utilities/CandybeanLogger.java

```diff
@@ -20,20 +21,6 @@ public CandybeanLogger(String name) {
 	}
 
 	/**
```

Comment:
Removed this function since it makes more sense to me to call iterator.remove in the original function, which leaves function with only 3 lines left, so I removed it.